test/ethers/shouldWorkWithEthers.ts

```diff
@@ -131,10 +126,20 @@ export function shouldWorkWithEthers({
       let networks: NetworksConfig
 
       before('obtain addresses from artifacts', async () => {
-        const { gnosisSafe, cpkFactory, multiSend, defaultCallbackHandler } = contracts
+        const { gnosisSafe, gnosisSafe2, cpkFactory, multiSend, defaultCallbackHandler } = contracts
 
         networks = {
           [(await signer.provider.getNetwork()).chainId]: {
+            masterCopyAddressVersions: [
+              {
+                address: gnosisSafe.address,
+                version: '1.2.0'
+              },
+              {
+                address: gnosisSafe2.address,
+                version: '1.1.1'
+              }
+            ],
             masterCopyAddress: gnosisSafe.address,
```

Comment:
I'd remove `masterCopyAddress` here because it actually overrides the `masterCopyAddressVersions` from what I can tell from the code. Of course, that would require making the `masterCopyAddress` field optional in the entry type.